src/coreclr/jit/gentree.cpp

```diff
@@ -22517,6 +22519,24 @@ GenTreeHWIntrinsic* Compiler::gtNewScalarHWIntrinsicNode(
                            /* isSimdAsHWIntrinsic */ false, op1, op2, op3);
 }
 
+GenTreeLclVar* Compiler::gtNewLclvForMultiRegIntrinsicNode(GenTreeHWIntrinsic* intrinsicNode, CORINFO_SIG_INFO* sig)
```

Comment:
This seems duplicative with `impAssignMultiRegTypeToVar`.

Reply:
That method is only defined under `FEATURE_MULTIREG_RET`, which is false under Windows x64. Multi-reg intrinsic wasn't treated as multi-reg return. It becomes quite mess then.

Reply:
You can just delete the `#ifdef`. Before this change, `FEATURE_MULTIREG_RET` was precisely equal to "supports multi-reg nodes in general".